muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/pedagogical-support-form/workspace-select.tsx

```diff
@@ -63,7 +63,7 @@ const WorkspaceSelect: React.FC<WorkspaceSelectProps> = (props) => {
       className="react-select-override react-select-override--hops"
       classNamePrefix="react-select-override"
       isClearable
-      placeholder="Etsi työtiloja..."
+      placeholder="Etsi kursseja..."
       options={options}
```

Comment:
Probably not scope of this pr, but is this fixed missing locale fixed in some other branch or?

Reply:
Currently it seems that the whole pedagogy form is not localized, same thing applies to hops form.